models/core-java/src/main/java/org/commonjava/indy/model/core/AccessChannel.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2011 Red Hat, Inc. (jdcasey@commonjava.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.model.core;
+
+/**
+ * Enumeration to distinguish between different access channels to stores.
+ *
+ * @author pkocandr
+ */
+public enum AccessChannel
```

Comment:
I suspect we may need to refactor this someday into either a string or maybe an interface. If we pull out the maven-specific parts of Indy into an add-on, then having the MAVEN_REPO access channel doesn't make sense in the core. GENERIC_PROXY doesn't make that much sense now, though it's a small thing. I'm not sure we have any more detail to add to an AccessChannel, so maybe just having a few string constants in different add-ons would be better? Then, we could make PNC simply log&drop things it doesn't recognize.